src/routes/public/organizations.js

```diff
@@ -23,9 +39,13 @@ exports.register = function (server, options, next) {
         }
       },
       validate: {
-        headers
+        headers,
+        query: Joi.object({
+          page: Joi.number().integer().positive().description('Page number, starts from 1'),
+          limit: Joi.number().integer().positive().description('Users per page')
```

Comment:
As both page and limit should be >= 1, i would replace positive with something like `.min(1)`. positive() allows also 0 values